packages/foundry/README.md

```diff
@@ -3,62 +3,66 @@
 A CLI to create new Workleap projects non-interactively. See `@workleap/foundry --help`:
 
 ```cmd
-Usage: foundry [options] [command]
+Usage: @workleap/foundry [options] [command]
 
 Foundry-CLI generator module
 
 Options:
-  -V, --version               output the version number
-  -h, --help                  display help for command
+  -V, --version                        output the version number
+  -h, --help                           display help for command
 
 Commands:
-  host-application [options]  use the host-application template
-  remote-module [options]     use the remote-module template
-  static-module [options]     use the static-module template
-  help [command]              display help for command
+  generate-host-application [options]  use the host-application template
+  generate-remote-module [options]     use the remote-module template
+  generate-static-module [options]     use the static-module template
+  help [command]                       display help for command
 ```
 
 ## Commands
 
-### host-application
+### generate-host-application
 
-Add and render the template from [host-application](https://github.com/workleap/wl-foundry-cli/tree/main/templates/host-application).
+Add and render the template
```

Comment:
It's about "cloning the template locally" (add) and "replace the text in the template" (render).

It can also be:
`Apply template and render content`
`Instantiate template and render output`
`Render template and update files`

I asked a friend for suggestions:
![image](https://user-images.githubusercontent.com/4323800/234378203-6044c4a7-0aca-4599-9559-71e5d234953a.png)

Reply:
I finally used a more verbose approach:
"Use the [host-application](https://github.com/workleap/wl-foundry-cli/tree/main/templates/host-application) template as a base to customize with these options:"